src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -3051,4 +3051,29 @@ namespace ts.projectSystem {
             service.checkNumberOfProjects({ externalProjects: 1 });
         });
     });
+
+    describe("maxNodeModuleJsDepth for inferred projects", () => {

```

Comment:
can you have a test that actually verifies the value of `maxModuleDepth` (i.e. that we are not looking into folders that are located deeper)